### PR TITLE
AVRO-2030: Link to Google Books Chapter

### PR DESCRIPTION
**Pull request by kojiromike**
### Jira

- [x] My PR addresses [AVRO-2030](https://issues.apache.org/jira/browse/AVRO-2030/)
- [x] My PR references it in the title.
- [x] My PR does not add any dependencies.

### Tests

- [x] My PR adds no tests because it is documentation only.

### Commits

- [x] My commits all reference Jira issues in their subject lines.
- [x] My commits follow the guidelines from "[How to write a good git commit message](https://chris.beams.io/posts/git-commit/)"

### Documentation

- [x] This PR updates the spec page on the website.

### CODE REVIEW

File: doc/src/content/xdocs/spec.xml [hunk 1]
```diff
@@ -1404,10 +1404,11 @@ void initFPTable() {
 }
         </source>
 
-        <p> Readers interested in the mathematics behind this
-        algorithm may want to read <a
-        href="https://www.scribd.com/fb-6001967/d/84795-Crc">this book
-        chapter.</a> (Unlike RFC-1952 and the book chapter, we prepend
+        <p>Readers interested in the mathematics behind this
+          algorithm may want to read
+        <a href="https://books.google.com/books?id=XD9iAwAAQBAJ&amp;pg=PA319"
```
Comment (RyanSkraba) [inline doc/src/content/xdocs/spec.xml]: If this is meant to link to the specific chapter, it doesn't work for me!  I do get the Google books reference to the book however.

Comment (kojiromike) [reply doc/src/content/xdocs/spec.xml]: Hah, good old Google, giving different results to different viewers.

Comment (kojiromike) [reply doc/src/content/xdocs/spec.xml]: I'm not really sure what better to do here -- it's not clear that the online PDFs of the book I could find were legitimate and freely licensed, so I guess I can either link to the book and let the interested folks acquire it via their own means, or I can remove the link entirely.

Comment (RyanSkraba) [reply doc/src/content/xdocs/spec.xml]: > let the interested folks acquire it via their own means

That's likely the best solution!  I'll comment on the JIRA for posterity!

**Pull request merged**